colour/models/rgb/transfer_functions/fuji_flog.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+"""
+Fuji F-Log Log Encoding
+============================
+
+Defines the *Fuji F-Log* log encoding:
+
+-   :func:`colour.models.log_encoding_FLog`
+-   :func:`colour.models.log_decoding_FLog`
+
+See Also
+--------
+`RGB Colourspaces Jupyter Notebook
+<http://nbviewer.jupyter.org/github/colour-science/colour-notebooks/\
+blob/master/notebooks/models/rgb.ipynb>`_
+
+References
+----------
+-   :cite:`FujiVer1` : F-Log Data Sheet Ver.1.0.
+    Retrieved from https://www.fujifilm.com/support/digital_cameras/\
+software/lut/pdf/F-Log_DataSheet_E_Ver.1.0.pdf
+"""
+
+from __future__ import division, unicode_literals
+
+import numpy as np
+
+from colour.models.rgb.transfer_functions import full_to_legal, legal_to_full
+from colour.utilities import Structure, as_float, from_range_1, to_domain_1
+
+__author__ = 'Colour Developers'
+__copyright__ = 'Copyright (C) 2013-2019 - Colour Developers'
+__license__ = 'New BSD License - http://opensource.org/licenses/BSD-3-Clause'
+__maintainer__ = 'Colour Developers'
+__email__ = 'colour-science@googlegroups.com'
+__status__ = 'Production'
+
+__all__ = ['FLOG_CONSTANTS', 'log_encoding_FLog', 'log_decoding_FLog']
+
+FLOG_CONSTANTS = Structure(
+    cut1=0.00089, cut2=0.100537775223865, a=0.555556, b=0.009468, c=0.344676,
+    d=0.790453, e=8.735631, f=0.092864)
+"""
+*Fuji F-Log* colourspace constants.
+
+FLOG_CONSTANTS : Structure
+"""
+
+
+def log_encoding_FLog(L_in,
+                      bit_depth=10,
+                      out_legal=False,
+                      in_reflection=True,
```

Comment:
Do we need support for that? Seems like the reference does not specifically mention it.